### PR TITLE
amount: update handle amount in checkout

### PR DESCRIPTION
**Pull request by allangrds**
## Description
Now the checkout calculate the amount by two ways:

* Getting `amount` prop from `transaction`
* Calc `amount` by sum and multiplication the items of cart.

Closes https://github.com/pagarme/mercurio/issues/231 

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

**Comment by allangrds** [comment]
Vou bolar um texto e colocar na issue do Mercurio

File: src/store.js [hunk 1]
```diff
@@ -31,7 +32,8 @@ export default ({
       },
       transactionValues: {
         ...transaction,
-        finalAmount: prop('amount', transaction),
+        amount: calcAmount(cart, shipping, transaction),
+        finalAmount: calcAmount(cart, shipping, transaction),
```
Comment (thiagommedeiros) [inline src/store.js]: Coloca esse `calcAmount` em uma `const`, assim vc nao precisa chamar ele duas vezes.

Comment (allangrds) [reply src/store.js]: Pronto @thiagommedeiros.

Veja novamente @augusto-jm-amaral, pf.

**Comment by allangrds** [comment]
@thiagommedeiros PR modificado!

**Pull request merged**